apps/emqx_authz/src/emqx_authz_app.erl

```diff
@@ -28,7 +28,7 @@
 -export([start/2, stop/1]).
 
 start(_StartType, _StartArgs) ->
-    ok = mria_rlog:wait_for_shards([?ACL_SHARDED], infinity),
+    ok = emqx_authz_mnesia:init_mria(),
```

Comment:
Nit: authz_*mnesia*:init_*mria* sounds a bit inconsistent. Just `init()`, perhaps?

Reply:
Let's choose `init_tables`, `init` is already taken by authz behaviour.